textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
TestConstants contains data for testing that everyone can use. But more likely each team has its own test data. So is it better to let every team put their own data in TestConstants.java, or to let every team keep their own test data inside their own test folder?